### PR TITLE
QA fixes

### PR DESCRIPTION
**Pull request by dianavicol**
- Fixed the restful messages pci issues

### CODE REVIEW

File: BankNetSP/develop/scripts/Process Incoming Message.js [hunk 1]
```diff
@@ -217,11 +217,13 @@ function set2(value) {
             break;
         default:
             card.setPan(value);
-            //card.setPanMasked(value);
+            card.setPanMasked(value);
```
Comment (fsemerdzhieva) [inline BankNetSP/develop/scripts/Process Incoming Message.js:220]: Why is this added here?

File: BankNetSP/develop/scripts/Process Incoming Message.js [hunk 2]
```diff
@@ -217,11 +217,13 @@ function set2(value) {
...
             try {
-                var sid = issuing.readSIDForPAN(txn, value);
+                //deep copy the value, otherwise it is wiped
+                var newValue = JSON.parse(JSON.stringify(value));
```
Comment (fsemerdzhieva) [inline BankNetSP/develop/scripts/Process Incoming Message.js:223]: Where is this wiped?

Comment (dianavicol) [reply BankNetSP/develop/scripts/Process Incoming Message.js:223]: SID manager class, but the changes are not merged yet

File: BankNetSP/develop/scripts/Process Incoming Message.js (hunk 1 above)
Comment (dianavicol) [reply BankNetSP/develop/scripts/Process Incoming Message.js]: Haven't noticed this on merging, my bad

**Review by fsemerdzhieva — APPROVED** [review]
Thanks Diana
